1.0/docs/devguide/data-binding.md

```diff
@@ -268,13 +268,13 @@ When you configure a declared property with the `notify` flag set to `true`,
     `this.firstName` will fire `first-name-changed`. Listeners will receive an event object 
     whose `e.detail.value` attribute has the changing property's new value.
 
-When using a {{site.project_title}} element with other elements or frameworks, you can 
-manually attach an <code>on-<var>property</var>-changed</code> listener to an element to
```

Comment:
note that this was a bug. should have been `property-changed`, not `on-property-changed`.